styles.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&display=swap');
+
+#toolbar {
+  display: flex;
+  position: fixed;
+  height: 35px;
+  width: 920px;
+  padding: 10px;
+}
+
+#logo {
+  color: #6070ff;
+  margin-right: 125px;
+  font-size: large;
+  font-weight: bold;
+}
+
+.nav-item {
+  display: flex;
+  margin-left: 10px;
+  font-family: 'Poppins', sans-serif;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  height: 740px;
```

Comment:
- [ ] [OPTIONAL] Please remove a fixed height as well, instead use padding-bottom to control the space after the social media icons. As you can see below that the space after the social media icons in your design is too much due to the fixed height. 

👇 Kindly fix this. _(This is marked as optional because the last reviewer did not mention it. But it is highly recommended that you implement it.)_

## The figma Design: Space after social media icon
![image](https://user-images.githubusercontent.com/84156714/185714230-e3e8afc0-f1de-4116-9ec7-0e5dab245f0f.png)
## Your design
![image](https://user-images.githubusercontent.com/84156714/185714147-bf6233e9-0063-4dca-b18b-9a1159891712.png)

Reply:
I appreciate this suggestion as well but since it is optional I will skip it now because I am running out of time. But in the future, I will consider the suggestion.